sass/tji/_tji.scss

```diff
@@ -13,7 +13,11 @@
 --------------------------------------------------------------*/
 @import "general";
 
+/*--------------------------------------------------------------
+## Explore the Data Styles
+--------------------------------------------------------------*/
+@import "data";
 /*--------------------------------------------------------------
 ## Helpers
 --------------------------------------------------------------*/
-@import "helpers";
+@import "helpers";
```

Comment:
Can we just take all the stuff in `helpers` and put it in `data`? Then delete `helpers`? I don't see much of a need for two separate files....